### PR TITLE
Fix/73885 email event dates

### PR DESCRIPTION
**Pull request by elimn**
Ticket: https://central.tri.be/issues/73885

Deprecated "start_date" filter and replaced it with a general "date" one, then swapped out our displaying of the start date for the full date.

### CODE REVIEW

**Comment by tr1b0t** [comment]
Status|Message|Line
:-----:|-----|-----
:x: | You must have a space after a closing parenthesis | [email.php:353](https://github.com/moderntribe/event-tickets/blob/6a919297efe83f95228bdcf4645f89c82dcb6728/src/views/tickets/email.php#L353)
:warning: | Expected next thing to be an escaping function (see Codex for 'Data Validation'), not '$event_date' | [email.php:355](https://github.com/moderntribe/event-tickets/blob/6a919297efe83f95228bdcf4645f89c82dcb6728/src/views/tickets/email.php#L355)

> via jenkins/codesniffer-phpcs

File: src/views/tickets/email.php [hunk 1]
```diff
@@ -339,9 +350,9 @@
 													<h2 style="color:#0a0a0e; margin:0 0 10px 0 !important; font-family: 'Helvetica Neue', Helvetica, sans-serif; font-style:normal; font-weight:700; font-size:28px; letter-spacing:normal; text-align:left;line-height: 100%;">
 														<a style="color:#0a0a0e !important" href="<?php echo esc_url( $event_link ); ?>"><?php echo $event->post_title; ?></a>
 													</h2>
-													<?php if ( ! empty( $start_date ) ): ?>
+													<?php if ( ! empty( $event_date ) ): ?>
```
Comment (ggwicz) [inline src/views/tickets/email.php]: add a space between `)` and `:`

File: src/views/tickets/email.php [hunk 2]
```diff
@@ -286,21 +286,32 @@
...
 				/**
-				 * Filters whether or not the event start date should be included in the ticket email
+				 * Filters whether or not the event date should be included in the ticket email.
+				 *
+				 * @since TBD
+				 *
+				 * @var bool Include event date? Defaults to false.
+				 * @var int  Event ID
+				 */
+				$include_event_date = apply_filters( 'event_tickets_email_include_event_date', false, $event->ID );
```
Comment (borkweb) [inline src/views/tickets/email.php]: Let's take this opportunity to switch the filter name to our event-tickets naming convention:

```
tribe_tickets_email_include_event_date
```

**Comment by elimn** [comment]
Good calls @ggwicz and @borkweb. I made the changes.

**Comment by tr1b0t** [comment]
Status|Message|Line
:-----:|-----|-----
:warning: | Expected next thing to be an escaping function (see Codex for 'Data Validation'), not '$event_date' | [email.php:355](https://github.com/moderntribe/event-tickets/blob/2cb60164b6c1a1a02d0f3856d4730389c83cdcba/src/views/tickets/email.php#L355)

> via jenkins/codesniffer-phpcs

**Pull request merged**